projects/plugins/jetpack/extensions/blocks/pinterest/test/fixtures/jetpack__pinterest__deprecated-1.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:jetpack/pinterest {"url":"https://www.pinterest.com.au/bakingamoment/bread-recipes/"} -->
+<a href="https://www.pinterest.com.au/bakingamoment/bread-recipes/" class="wp-block-jetpack-pinterest">https://www.pinterest.com.au/bakingamoment/bread-recipes/</a>
+<!-- /wp:jetpack/pinterest -->
```

Comment:
This will need to actually match the content generated by the save function of the deprecation. 

That is, it should be wrapped in the `<div className={ className }></div>`. Otherwise I don't believe it actually gets processed by the deprecation we are intending to test with this fixture.

Reply:
Thanks for catching that. I remember seeing it, but totally forgot to change it. 👍